src/partials/products.html

```diff
@@ -1,4 +1,4 @@
-<!-- <link rel="stylesheet" href="../sass/_products.scss" /> -->
+<link rel="stylesheet" href="../sass/_products.scss" />
```

Comment:
skoro nam dziala npm run dev i widzimy strone na localhost to to jest do wywalenia, strone piszemy na podgladzie localhost, zeby na final na pewno dobrze wygladała